app/controllers/idv/phone_controller.rb

```diff
@@ -113,13 +113,19 @@ def failure_url(reason)
 
     def async_state_done(async_state)
       form_result = step.async_state_done(async_state)
+
+      configured_phones = MfaContext.new(current_user).phone_configurations.map(&:phone)
```

Comment:
Do we need to make sure these are formatted too?

https://github.com/18F/identity-idp/blob/a4517f9bcf54e1d2a94f6baac269b2ff6c996580/app/forms/new_phone_form.rb#L84-L86 formats all of them

Reply:
In the old ConfirmationsController, neither needed to be explicitly formatted, but when I moved it here, the applicant's phone's format didn't match the configured phones. I'll take a closer look at this.

Reply:
It's possible the old version had a bug there then. I don't think we store phone numbers in a consistent format, so checking for existing numbers requires the above :/

Reply:
Gotcha. See fe54311ee